applications/StructuralMechanicsApplication/tests/cl_test/SmallDeformationPlasticity/small_deformation_plasticity_test_parameters.json

```diff
@@ -0,0 +1,121 @@
+                "interval"        : [0.0,"End"]
+            }
+        }],
+        "json_check_process" : [
+        {
+            "python_module"   : "from_json_check_result_process",
+            "kratos_module" : "KratosMultiphysics",
+            "help"                  : "",
+            "process_name"          : "FromJsonCheckResultProcess",
+            "Parameters"            : {
+                "gauss_points_check_variables" : ["UNIAXIAL_STRESS"],
+                "input_file_name"              : "cl_test/SmallDeformationPlasticity/small_deformation_plasticity_test_results.json",
+                "model_part_name"              : "Parts_Parts_Auto1",
+                "time_frequency"               : 1.0
+            }
+        }
+        ],
+        "loads_process_list"       : [],
+        "list_other_processes"     : []
+    },
+    "_json_output_process" : [
+    {
+        "python_module"   : "json_output_process",
+        "kratos_module" : "KratosMultiphysics",
+        "help"                  : "",
+        "process_name"          : "JsonOutputProcess",
+        "Parameters"            : {
+            "gauss_points_output_variables" : ["UNIAXIAL_STRESS"],
+            "output_file_name"              : "cl_test/SmallDeformationPlasticity/small_deformation_plasticity_test_results.json",
+            "model_part_name"               : "Parts_Parts_Auto1",
+            "time_frequency"                : 1.0
+        }
+    }
+    ],
+    "_output_processes" : {
+        "gid_output" : [{
+            "python_module" : "gid_output_process",
+            "kratos_module" : "KratosMultiphysics",
+            "process_name"  : "GiDOutputProcess",
+            "help"          : "This process writes postprocessing files for GiD",
+            "Parameters"    : {
+                "model_part_name"        : "Structure.computing_domain",
+                "output_name"            : "small_deformation_plasticity_test",
+                "postprocess_parameters" : {
+                    "result_file_configuration" : {
+                        "gidpost_flags"       : {
+                            "GiDPostMode"           : "GiD_PostBinary",
+                            "WriteDeformedMeshFlag" : "WriteDeformed",
+                            "MultiFileFlag"         : "SingleFile"
+                        },
+                        "file_label"          : "step",
+                        "output_control_type" : "step",
+                        "output_frequency"    : 1,
+                        "body_output"         : true,
+                        "node_output"         : false,
+                        "skin_output"         : false,
+                        "plane_output"        : [],
+            "nodal_results"       : ["DISPLACEMENT"],
+            "gauss_point_results" : ["GREEN_LAGRANGE_STRAIN_TENSOR","CAUCHY_STRESS_TENSOR","UNIAXIAL_STRESS","PLASTIC_DISSIPATION"]
+                    },

```

Comment:
I would like to include here "EQUIVALENT_PLASTIC_STRAIN"

Reply:
We should register EQUIVALENT_PLASTIC_STRAIN for post-processing

Reply:
I can add it